examples/dao/assets/proposal-lsig.py

```diff
@@ -0,0 +1,11 @@
+from pyteal import *
+
+def proposal_lsig():
+    """
+    Represents Proposal Lsig (a contract account owned by the proposer)
+    """
```

Comment:
we should have conditions here. What about:
1. algo and asset withdraw must be done by "owner" (template value)
2. opt-in only by the owner
3. only owner can close reminder to
4. ... maybe more?

This way we assure only owner can manage the assets of the lsig.

Reply:
Ok, i didn't do it before because we don't control this account (it belongs to the proposer). We can provide a template with these checks, and recommend to use it.

Reply:
yes, that's the idea. We also need to document / explain it.